glambie/notebooks/demo-query-submitted-data.ipynb

```diff
@@ -0,0 +1,635 @@
+{
```

Comment:
There are good reasons to avoid notebooks in repos. They're tough to maintain, they're large, and they tend to proliforate. I think having 1 notebook in a repo for the purpose of basic education is ok. This folder implies it's going to have lots of notebooks which we should definitely avoid. Is the intention to have lots of notebooks? Can this folder be removed and it simply be called usage_examples.ipynb. Is it essential that it's a notebook - i.e. is plain python sufficient? Could we bake this stuff into auto-generated doc-string documentation per Specklia.